layout/_partials/footer.swig

```diff
@@ -34,9 +34,13 @@
   {% endif %}
 </div>
 
+{% if theme.seo %}
+  {% set nofollow = ' rel="external nofollow"' %}
```

Comment:
Just like this.

Reply:
OK, I will adopt it in `sidebar.swig`

Reply:
No, there is no needed because there are single conditions.